examples/tutorials/curation/plot_1_automated_curation.py

```diff
@@ -0,0 +1,248 @@
+#    )
+#
+# If you run this locally you will receive an error:
+#
+# .. code-block::
+#
+#     UntrustedTypesFoundException: Untrusted types found in the file: ['sklearn.metrics._classification.balanced_accuracy_score', 'sklearn.metrics._scorer._Scorer', 'sklearn.model_selection._search_successive_halving.HalvingGridSearchCV', 'sklearn.model_selection._split.StratifiedKFold']
+#
+# This is a security warning. Sharing models, with are Python objects, is complicated.
+# We have chosen to use the `skops format <https://skops.readthedocs.io/en/stable/>`_, instead
+# of the common but insecure ``.pkl`` format (read about ``pickle`` security issues
+# `here <https://lwn.net/Articles/964392/>`_). While unpacking the ``.skops`` file, each function
+# is checked. Ideally, skops should recognise all `sklearn`, `numpy` and `scipy` functions and
+# allow the object to be loaded if it only contains these (and no unkown malicious code). But
+# when ``skops`` it's not sure, it raises an error. Here, it doesn't recognise
+# ``['sklearn.metrics._classification.balanced_accuracy_score', 'sklearn.metrics._scorer._Scorer',
+# 'sklearn.model_selection._search_successive_halving.HalvingGridSearchCV',
+# 'sklearn.model_selection._split.StratifiedKFold']``. Taking a look, these are all functions
+# from `sklearn`, and we can happily add them to the ``trusted`` functions to load:
+#
+# .. code-block::
+#
+#     model, model_info = sc.load_model(
+#         model_name = 'noise_neuron_model.skops',
+#         repo_id = "AnoushkaJain3/curation_machine_learning_models",
+#         trusted = ['sklearn.metrics._classification.balanced_accuracy_score', 'sklearn.metrics._scorer._Scorer', 'sklearn.model_selection._search_successive_halving.HalvingGridSearchCV', 'sklearn.model_selection._split.StratifiedKFold']
+#     )
+#
+# As ``skops`` continues to be developed, we hope more of these functions will be :code:`trusted`
+# by default.
+#
+# If you unequivocally trust the model (e.g. if you have created it), you can bypass this security
+# step by passing ``trust_model = True`` to the ``load_model`` function.
+#
+# In general, you should be cautious when downloading ``.skops`` files and ``.pkl`` files from repos,
+# especially from unknown sources.
+#
+#
+# Directly applying a sklearn Pipeline
+# ------------------------------------
+#
+# Instead of using ``HuggingFace`` and ``skops``, you might have another way of receiving a sklearn
+# pipeline, and want to apply it to your sorted data.
+
+from spikeinterface.curation.model_based_curation import ModelBasedClassification
+
+model_based_classification = ModelBasedClassification(sorting_analyzer, model)
+labels = model_based_classification.predict_labels()
+labels
+
+##############################################################################
+# Using this, you lose the advantages of the model metadata: the quality metric parameters
+# are not checked and the labels are not converted their original human readable names (like
```

Comment:
Could go into a bit more detail here, what does 'quality metric parameters are not checked' mean, what are the implications? For the labels, this doesn't seem like much of a problem for the good/bad case. Are there times it is very confusing e.g. 4-5 possible labels good bad mua noise etc?